### PR TITLE
Added meta tags, title, and favicon

### PR DESCRIPTION
**Pull request by landryroni**
Changes:
- Updated the title tag
- Added images for favicon and meta banner
- Added meta tags, including OG and Twitter tags
- Added favicon link tag

Test cases checked
- Verified the title showing up

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,9 +2,22 @@
...
+        <meta name="robots" content="index, follow">
+        <meta property="og:title" content="BYC Careers | Build a Successful Caree in Tech">
+        <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+        <meta property="og:image" content="byc_meta.jpg">
+        <meta name="twitter:title" content="BYC Careers | Build a Successful Caree in Tech">
+        <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+        <meta name="twitter:image" content="byc_meta.jpg">
         <title>
-            My Frist web page
+            BYC Careers | Build a Successful Caree in Tech
```
Comment (krylovaye2017) [inline src/index.html]: Make the title longer please, maybe atleast 24 characteres

File: src/index.html [hunk 2]
```diff
@@ -2,9 +2,22 @@
 <html>
 
     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Learn about job openings at BYC and submit your application">
```
Comment (krylovaye2017) [inline src/index.html]: This can be improved a bit

**Review by krylovaye2017 — CHANGES_REQUESTED** [review]
Request some changes

File: src/index.html (hunk 2 above)
Comment (landryroni) [reply src/index.html]: I think this is fine as it is.

**Review by krylovaye2017 — APPROVED** [review]

**Pull request merged**